python-sdk/pyproject.toml

```diff
@@ -94,6 +94,11 @@ mssql = [
     "apache-airflow-providers-microsoft-mssql>=3.2",
 ]
 
+duckdb = [
+    "airflow-provider-duckdb",
+    "duckdb-engine"
```

Comment:
should duckdb-engine be a dependency of the SDK or of the duckdb airflow provider?

Reply:
It shouldn't unless we need to patch the provider temporarily. Ideally, we should only depend on the provider.

Reply:
Yes duckdb-engine should have this dependency. It's just a temporary solution. I have already create a PR for this: https://github.com/astronomer/airflow-provider-duckdb/pull/2

Reply:
This can be removed once this change is released on providers side

Reply:
removed now: https://github.com/astronomer/astro-sdk/pull/1695/commits/a230d7316d24af6a98ed444557a21c4da4ae4609